Controllers/ToppingsController.js

```diff
@@ -5,9 +5,34 @@ const ObjectId = require("mongodb").ObjectId;
 
 const getAllToppings = (req, res) => {
   DbService.connectToDb(async (db) => {
-    const toppings = await ToppingsService.getAllToppings(db);
-    res.json(toppings);
+    const result = await ToppingsService.getAllToppings(db);
+    let jsonRes = ResponsesService.successful()
+    jsonRes.data = result;
+    return res.json(jsonRes);
   });
 };
 
+const updateTopping = (req, res) => {
+  const data = {
+    id: ObjectId(req.body.id),
+    wins: req.body.wins,
+    battles: req.body.battles,
+  };
+  data.winPercent = Number((100*data.wins/data.battles).toFixed(1));
+
+  DbService.connectToDb(async (db) => {
+    const result = await ToppingsService.updateTopping(db, data);
+    if(result.modifiedCount == 1) {
+      const result2 = await ToppingsService.getToppingById(db, data.id);
+      let jsonRes = ResponsesService.successful();
+      jsonRes.data = result2;
+      return res.json(jsonRes);
+    } else {
+      let jsonRes = ResponsesService.unsuccessful();
+      return res.json(jsonRes);
+    }
```

Comment:
Since there is a return statement in side the if statement, you actually don't need the else condition. The unsuccessful stuff can just sit outside the if statement.

This is called early returning and helps keep code a bit simpler